src/export/digitalPlanning/model.ts

```diff
@@ -301,20 +303,25 @@ export class DigitalPlanning {
   }
 
   private getOwners(): Owners[] {
+    const applicantInterest = this.passport.data?.[
+      "applicant.ownership.interest"
+    ]?.[0] as string;
+    const ownersListItems = this.passport.data?.[
+      "property.ownership.owner"
+    ] as unknown as Array<Owners>; // same passport field used for both/either List schemas
```

Comment:
Defining these variables upfront just makes following conditions easier to read, no functional/logical changes here !